### PR TITLE
Prepare for migration from Heroku

### PR DESCRIPTION
**Pull request by theseal**
Redirect /calendar and /average to custom set domain.

How to handle the static site? @kokarn any idea? Feel free to commit to this branch directly.

### CODE REVIEW

**Comment by kokarn** [comment]
Something like this should work

File: app.js [hunk 1]
```diff
@@ -6,19 +6,23 @@ const teamData = require( './modules/teamdata' );
...
 app.set('port', (process.env.PORT || 5000));
 app.set('redirect_to', (process.env.REDIRECT_TO || false));
-app.use(express.static('static'));
+
+app.use((req, res, next) => {
+    if ( app.get('redirect_to') && req.hostname !== app.get('redirect_to')){
+        res.redirect(301,'https://' + app.get('redirect_to') + req.originalUrl);
+    }
+
+    next();
```
Comment (theseal) [inline app.js:15]: What does `next` in this context? Next of what? 😀

Comment (kokarn) [reply app.js:15]: Next is what to do _after_ that specific middleware has run. So in this case it would be "proceed to the next handler"

Comment (theseal) [reply app.js:15]: That is what I thought. Unfortunately the code doesn't work if the client is redirected:
```
➜  hockey-mchockeyface git:(redirect-from-heroku) ✗ REDIRECT_TO=kubernetes.docker.internal node ./app.js
Hockey McHockeyFace ready to serve on port 5000
node:internal/errors:484
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:393:5)
    at ServerResponse.setHeader (node:_http_outgoing:644:11)
    at ServerResponse.header (/Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/response.js:794:10)
    at /Users/jocar/git/hockey-mchockeyface/app.js:72:7
    at Layer.handle [as handle_request] (/Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jocar/git/hockey-mchockeyface/node_modules/express/lib/router/index.js:346:12) {
  code: 'ERR_HTTP_HEADERS_SENT'
}

Node.js v19.0.1

```

I think we need to return the client as soon possible after the redirect. Is an old `return` good enough (seems to work at least) or is there a better way to do that in `express`?

Comment (kokarn) [reply app.js:15]: Return is fine, and you can return the "redirect" so it looks a bit nicer IMO.

**Pull request merged**